R/asis-nodes.R

```diff
@@ -320,3 +320,57 @@ protect_footnotes <- function(body, ns = md_ns()) {
   copy_xml(body)
 }
 #nocov end
+# CURLY ------------------
+find_curly <- function(body, ns) {
+   i <- ".//md:text[not(@asis) and contains(text(), '{')]"
+   xml2::xml_find_all(body, i, ns = ns)
+}
+
+fix_curly <- function(curly) {
+  char <- as.character(curly)
+  curlies <- regmatches(char, regexpr("\\{.*?\\}", char))
+  for (curl in curlies) {
+    char <- sub(
+      curl,
+      sprintf("</text><text asis='true' curly='true'>%s</text><text>", curl),
```

Comment:
I also added the curly attribute because it might be useful to differentiate the asis node -- is that a bad idea?

Reply:
Nope! I think it's in line with what we want to do. In fact, I don't think we need the `asis` node here because we do not necessarily need to protect things inside curlies, unless there is something I am missing.